docs/cluster-management/config.md

```diff
@@ -73,7 +76,7 @@ tags:  # <string>: <string> map of key/value pairs
 # see https://docs.cortex.dev/v/master/cluster-management/spot-instances for additional details on spot configuration
 spot: false
 
-# see https://docs.cortex.dev/v/master/guides/subdomain-https-setup for instructions on how to set up HTTPS for APIs
+# see https://docs.cortex.dev/v/master/guides/custom-domain for instructions on how to set up a custom domain
```

Comment:
I think we can simplify this by making a guide on how to add custom [domains](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-custom-domain-names.html) with API gateway which is easier!